src/reducers/bookings.ts

```diff
@@ -208,12 +227,12 @@ const aggregateUsedSlots = (
   })
 }
 
-// calculate used spaces at a especific moment
-const aggregateUsedSpaces = (
+// calculate used items at a especific moment
```

Comment:
```suggestion
// calculate used items at a specific moment
```

that was probably written by a Spanish speaking person :) (LagarSoft is Argentinia based?)

Reply:
[42e57e9](https://github.com/archilogic-com/room-booking/pull/26/commits/42e57e9f9c023cff8dc4b4cfacf59582612cace0)